Makefile

```diff
@@ -232,10 +231,10 @@ test-sim:
 	@VERSION=$(VERSION) go test -mod=readonly $(PACKAGES_SIM)
 
 test-e2e:
-	@VERSION=$(VERSION) OSMOSIS_E2E_UPGRADE_VERSION="v12" go test -mod=readonly -timeout=25m -v $(PACKAGES_E2E)
+	@VERSION=$(VERSION) OSMOSIS_E2E_UPGRADE_VERSION="v12" go test ./... -tags=e2e -mod=readonly -timeout=25m  
```

Comment:
I think we still want `-v $(PACKAGES_E2E)`. Currently, the entire test suite is run on `make test-e2e as opposed to only e2e test

Reply:
I changed this because `PACKAGES_E2E=$(shell go list ./... | grep '/e2e')` don't contain  package with build tag `e2e` (`github.com/osmosis-labs/osmosis/v10/tests/e2e`).

Reply:
When I run `go list ./... | grep '/e2e'` : 
```
github.com/osmosis-labs/osmosis/v10/tests/e2e/configurer
github.com/osmosis-labs/osmosis/v10/tests/e2e/configurer/chain
github.com/osmosis-labs/osmosis/v10/tests/e2e/configurer/config
github.com/osmosis-labs/osmosis/v10/tests/e2e/containers
github.com/osmosis-labs/osmosis/v10/tests/e2e/initialization
github.com/osmosis-labs/osmosis/v10/tests/e2e/initialization/chain
github.com/osmosis-labs/osmosis/v10/tests/e2e/initialization/node
github.com/osmosis-labs/osmosis/v10/tests/e2e/util
```

Reply:
Please consider amending to `go list -tags=e2e ./... | grep e2e`. That should help with including e2e package as well

Reply:
Yes, agree with @p0mvn here 👍

Reply:
Yes, thank for your suggestion. I changed it.